priompt/src/lib.ts

```diff
@@ -234,17 +230,19 @@ export function createElement(tag: ((props: BaseProps & Record<string, unknown>)
 	}
 }
 
-export function Fragment({ children }: { children: PromptElement[]; }): PromptElement {
-	// merge all the lists
-	return children.flat();
+export function Fragment({ children }: { children: PromptNode; }): PromptElement {
```

Comment:
does Fragment need to return a PromptElement or is it fine to return a PromptNode? Ideally we wouldn't need to introduce the scope here

Reply:
Ah I'll fix this as well. A `PromptNode` will work here.